src/app/profile.service.ts

```diff
@@ -8,23 +8,22 @@ import { Observable } from 'rxjs';
 })
 export class ProfileService {
 
-  private url1 = 'http://localhost:8765/interview-service/profiles/allProfiles';
-  private url2 = 'http://localhost:8765/interview-service/profiles/saveProfile';
+  private url1 = 'http://localhost:8765/interview-service/profiles';
```

Comment:
This was a pre-existing issue, but while we have the chance we should clean up this code a little bit. Previously, url2 wasn't used so having deleted it is good. But also now that the urls have changed, url1 can actually act as a base url. And our methods can define their specific url off of that. So for example, const url = baseUrl + "/1" or just const url = baseUrl if it is the same. This would make it easier to maintain in the future if the base url is ever changed. So please go ahead and make these changes while we're here.